Backend/Api/server.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const cors = require('cors');
+require('dotenv').config();
+const port = process.env.PORT;
+const app = express();
+
+app.use(cors());
+app.use(express.json());
+
+//User Router
+require('./Login/routes/user.route')(app);
```

Comment:
is this possible to use main router like v1/user/ . then we can follow the route with using router .